lib/modules/dosomething/dosomething_api/resources/reportback_item_resource.inc

```diff
@@ -158,7 +158,33 @@ function _reportback_item_resource_index($campaigns, $exclude, $status, $count,
 
   if (variable_get('rogue_collection', FALSE)) {
     $client = dosomething_rogue_client('v1');
-    $response = $client->getAllReportbacks($inputs);
+    $query_parts = explode("&", $_SERVER['REQUEST_URI']);
```

Comment:
@DFurnes I'm trying to get the page number that is set in `Reportback.js` on line 290 of this PR. Is this the best way to do it? I'm having a bit of trouble grabbing the page number. If it's easier to walk you through IRL what I was thinking, LMK!